packages/ws-worker/test/util/cli.test.ts

```diff
@@ -108,3 +108,19 @@ test('cli should handle array options correctly for env variables', (t) => {
 
   t.deepEqual(args.statePropsToRemove, ['prop1', 'prop2', 'prop3']);
 });
+
+test('cli should parse WORKER_CAPACITY from env', (t) => {
```

Comment:
I guess I don't have a problem with these two test, but is there any reason they were added? What is being tested, the names of the env vars?